homeassistant/components/minecraft_server/strings.json

```diff
@@ -0,0 +1,26 @@
+{
+    "config": {
+        "title": "Minecraft Server",
+        "step": {
+            "user": {
+                "title": "Link your Minecraft Server.",
+                "description": "Set up your Minecraft Server instance to allow monitoring.",
+                "data": {
+                    "name": "Name",
+                    "host": "Host Name",
+                    "port": "Port Number",
```

Comment:
"Host" is enought or "Host name"
"Port" is enought

Reply:
Done.